src/Modules/General/FoodbankTextComponent.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+const FoodbankTextComponent = ({ text ='', imageUrl = '', LinkUrl = '',linkText ='' }) => {
+	 
+    return (       
+            <div className="row">
+                <div className="col-1 item-icon d-flex align-items-center">
+                    <img alt={text} src={imageUrl} />
+                </div>
+                <div className="col-8 item-details">
+                    <p>{text}</p>
+                </div>
+                <div style={{"word-wrap": "break-word"}} className="col-3">
+                    <a href={LinkUrl} target="_blank"> {linkText}</a>
```

Comment:
Using target="_blank" without rel="noopener noreferrer" is a security risk
see- https://mathiasbynens.github.io/rel-noopener

Reply:
u can update 
from `<a href={LinkUrl} target="_blank"> {linkText}</a>`
 to `<a href={LinkUrl} target="_blank" rel="noopener noreferrer"> {linkText}</a> `

Reply:
Done added